### PR TITLE
add support to read Secret

### PR DESCRIPTION
**Pull request by usamaahmadkhan**

### CODE REVIEW

**Comment by stakater-user** [comment]
@usamaahmadkhan Yikes! You better fix it before anyone else finds out! [Build 1](https://jenkins.release.stakater.com/job/stakater/job/Xposer/job/PR-21/1/) has Failed!

**Comment by stakater-user** [comment]
@usamaahmadkhan Yikes! You better fix it before anyone else finds out! [Build 2](https://jenkins.release.stakater.com/job/stakater/job/Xposer/job/PR-21/2/) has Failed!

**Comment by stakater-user** [comment]
@usamaahmadkhan Image is available for testing. ``docker pull stakater/xposer:SNAPSHOT-PR-21-3``

**Comment by stakater-user** [comment]
@usamaahmadkhan Image is available for testing. ``docker pull stakater/xposer:SNAPSHOT-PR-21-4``

File: internal/pkg/controller/controller.go [hunk 1]
```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/stakater/Xposer/internal/pkg/constants"
 	"github.com/stakater/Xposer/internal/pkg/ingresses"
 	"github.com/stakater/Xposer/internal/pkg/routes"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```
Comment (aliartiza75) [inline internal/pkg/controller/controller.go]: is `v1` required?

Comment (usamaahmadkhan) [reply internal/pkg/controller/controller.go]: Yes. VSCode was removing it automatically. So it had to be put like that

**Pull request merged**